app/services/doc_auth/lexis_nexis/image_metrics_reader.rb

```diff
@@ -0,0 +1,29 @@
+module DocAuth
+  module LexisNexis
+    module ImageMetricsReader
```

Comment:
I'm fine with extracting this code from TrueIDResponse to a module.  Are you okay with the test coverage as is in true_id_response_spec.rb, or should we add more test coverage for the module?  (Also, I understand that the methods were private in TrueIdResponse, but what's the benefit of making the whole module private?)

Reply:
@eileen-nava , i think private here is for the methods, not the module? 

My main intention is to split out these data extraction code out, since they don't change unless the response structure changes, so moving parts will be concentrated in `TrueIDResponse`

Reply:
@eileen-nava , I thought about the testing too and believe current test has enough coverage.

Reply:
Ah, I should have said "making all of the methods in the module private" instead of "making the whole module private."  I've seen private methods make testing much harder, which is why I asked about it here.  I'm okay with leaving it as is. 👍🏻